lib/Controller/PaperHiveController.php

```diff
@@ -142,7 +142,7 @@ public function getPaperHiveBookDiscussionCount($dir, $filename) {
 			// Silently ignore discussions as this might indicate temporary unavailability
 			$disscussionCount = \count($paperHiveDiscussions['discussions']);
 		}
-		return new DataResponse($disscussionCount, Http::STATUS_OK);
+		return new DataResponse([$disscussionCount], Http::STATUS_OK);
```

Comment:
This is causing the unit tests to fail.... But `DataResponse` should take array/object in its constructor, not int

@phil-davis  should we fix the unit test or ?

Reply:
Everywhere else in apps `DataResponse` is being passed an array. So this change is correct.
Fix the unit tests.

Reply:
OK, looked again. This fix is good.